scenes/player/player_controller.gd

```diff
@@ -21,11 +21,11 @@ var actual_life: float = total_life
 @export var magnitude_parry_enemy:float = 50
 #
 ##Is_Light_form?
-var is_light_player: bool = true
+@export var is_light_player: bool = true
```

Comment:
It should not be exported. It should be private, encapsulated in player class, don't you think?